### ISSUE TITLE
No metadata found

### ISSUE THREAD

**Issue by rmadnantariq** [MattiaPandolfoVR/MetaPhage#41]
Hi, I am trying to run the tool but get this error message. I have used the newproject.py to create the .conf file I use that to alter my cluster file and get the following error message,

# conda environments:
#
base                  *  /home/crk_tgms5/miniconda
MetaPhage                /home/crk_tgms5/miniconda/envs/MetaPhage
PhageBoost-env           /home/crk_tgms5/miniconda/envs/PhageBoost-env
ProphET                  /home/crk_tgms5/miniconda/envs/ProphET
TraDIS                   /home/crk_tgms5/miniconda/envs/TraDIS
bacterialassembly        /home/crk_tgms5/miniconda/envs/bacterialassembly
cutadaptenv              /home/crk_tgms5/miniconda/envs/cutadaptenv
flye                     /home/crk_tgms5/miniconda/envs/flye

N E X T F L O W  ~  version 21.04.0
Launching `main.nf` [thirsty_brahmagupta] - revision: 22a1357f42

====================================================
 __  __      _        _____  _                      
|  \/  |    | |      |  __ \| |                     
| \  / | ___| |_ __ _| |__) | |__   __ _  __ _  ___ 
| |\/| |/ _ \ __/ _` |  ___/| '_ \ / _` |/ _` |/ _ \
| |  | |  __/ || (_| | |    | | | | (_| | (_| |  __/
|_|  |_|\___|\__\__,_|_|    |_| |_|\__,_|\__, |\___|
                                          __/ |     
                                         |___/      
====================================================

 Input:      /home/crk_tgms5/MetaPhage/datasets/Greg
 Metadata:   /home/crk_tgms5/MetaPhage/datasets/Greg/MetaPhage_c4g569fp_metadata
 Databases:  /home/crk_tgms5/MetaPhage/db
 Output:     /home/crk_tgms5/MetaPhage/MetaPhage_out
 Resources:  16 CPUs, 64 GB RAM
No metadata supplied! Metadata are mandatory for beta diversity and heatmaps plots!
If you are interested in these plots, simply provide a metadata .csv file using params.metadata in your config file!
For more informations, check the metadata paragraph on the wiki at https://github.com/MattiaPandolfoVR/MetaPhage#input_files
No such variable: ch_metadata_checker

 -- Check script 'main.nf' at line: 79 or see '.nextflow.log' file for more details

Any help would be much appreciated.

**Comment by MattiaPandolfoVR** [comment]
Hi, is the metadata file you provide a .csv file?

**Comment by MattiaPandolfoVR** [comment]
There was a typo in the newProject.py script, just fixed it. Please be sure that your metadata file is a .CSV with the first column header as "Sample" and be sure that the sample names in your metadata match the reads name

**Comment by rmadnantariq** [comment]
Thank you, I tried again with newProject.py script. Yes the sample header was Sample and I made sure the match the read names. [groups.csv](https://github.com/MattiaPandolfoVR/MetaPhage/files/8540665/groups.csv) but I still get the same error.

**Comment by MattiaPandolfoVR** [comment]
The name of your .csv file should end with "metadata.csv" . I'll modify this as it's not user friendly, thank you for pointing me this and let me know if it fixes the error!

**Comment by rmadnantariq** [comment]
No, that didn't solve it for me. When I generated the configuration file using the newProject.py it makes a symlink folder of the metadata file. Do I provide the path to the folder containing the .csv or symlink folder?
